src/assets/alignmentD3Tests.js

```diff
@@ -0,0 +1,376 @@
+export function getXAxisInfo(xAxis) {
+  return getAxisInfo(xAxis, getXBegin, getXEnd, getXTickLocation);
+}
+
+/**
+ * Get y coordinate of tick
+ * @function
+ * @param {HTMLElement} tick - HTMLElement g for given tick
+ * @returns {Number} pixel value of y translate coordinate
+ */
+var getYTickLocation = function(tick) {
+  return tick.getAttribute('transform').split(',')[1].split(')')[0];
+};
+
+/**
+ * Get x coordinate of tick
+ * @function
+ * @param {HTMLElement} tick - HTMLElement g for given tick
+ * @returns {Number} pixel value of x translate coordinate
+ */
+var getXTickLocation = function(tick) {
+  return tick.getAttribute('transform').split(',')[0].split('(')[1];
+};
+
+/**
+ * Convert one axis tick's label innerHTML from mm:ss to decimal
+ * @function
+ * @param {String} axisTick - innerHTML from text of a single tick. Ex: "37:00"
+ * @returns {Number} float
+ */
+var getTickValueMinutes = function(axisTick) {
+  return parseInt(axisTick.split(':')[0], 10) +
+    (parseInt(axisTick.split(':')[1], 10) / 60);
+};
+
+/**
+ * Convert feature data-yvalue attribute from date object to to decimal minutes
+ * @function
+ * @param {String} feature - HTMLElement circle (.dot)
+ * @returns {Number} float
+ */
+var getFeatureValueMinutes = function(feature) {
+  var value = feature.getAttribute('data-yvalue');
+
+  return new Date(value).getMinutes() +
+    (new Date(value).getSeconds() / 60);
+};
+
+/**
+ * Parse feature data-xvalue attribute
+ * @function
+ * @param {String} feature - HTMLElement circle (.dot)
+ * @returns {Number} integer
+ */
+var getFeatureValueYears = function(feature) {
+  var value = feature.getAttribute('data-xvalue');
+
+  return parseInt(value, 10);
+};
+
+/**
+ * Compare feature coordinate to tick[i] and tick[i+1] and return true if
+ * feature coordinate is greater than or equal to the coordinate for tick[i]
+ * and less than or equal to the coordinate for tick[i+1]
+ * @function
+ * @param {Number} featureCoord - x or y coordinate per feature per axis
+ * @param {Object} axis - Object constructed from getAxisInfo
+ * @param {Number} tickPxCur - Coordinate (x or y) for tick[i]
+ * @param {Number} tickPxNext - Coordinate (x or y) for tick[i+1]
+ * @returns {Boolean} true if feature position is between (inclusive) tick[i]
+ * and tick[i+1]
+ */
+var isFeatureAligned = function(featureCoord, axis, tickPxCur, tickPxNext) {
+  // Determine lower / higher coordinate size between two ticks
+  // This is to account for reverse tick order (see issue #70)
+  var minTickCoord = Math.min(tickPxCur, tickPxNext);
+  var maxTickCoord = Math.max(tickPxCur, tickPxNext);
+  // getMisalignmentCount checked whether the feature's attribute data-xvalue or
+  // data-yvalue is between (inclusive) two ticks' labeled values.
+  // If that is true, the feature's pixel coordinates should also be between
+  // (inclusive) the two ticks' axis coordinates.
+  return featureCoord >= (minTickCoord - 1) && featureCoord <= (maxTickCoord + 1)
```

Comment:
Could you please just document why you are adding and subtracting a pixel? Otherwise someone might try to fix it.